stablehlo/dialect/StablehloOps.td

```diff
@@ -1492,6 +1492,13 @@ def StableHLO_DynamicSliceOp: StableHLO_Op<"dynamic_slice",
     Extracts a sub-array from the input array at dynamic start_indices.
 
     See https://www.tensorflow.org/xla/operation_semantics#dynamicslice.
+
+    Example:
+
+    ```mlir
+    %0 = stablehlo.dynamic_slice %arg2, %arg3, %arg3, sizes = [1, 4] 
```

Comment:
#24 proposes to drop names altogether in situations when there's only one attribute (like this time). Can you document here (via a PR comment) on the rationale behind changing this to always provide attribute names?

Reply:
Fixed 24. Decided on `key = value` since it is clear, not too verbose, and allows unambiguous consistency between single-attr and multi-attr ops. It is far more scalable at minimal cost.